include/librealsense2/dds/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 # License: Apache 2.0. See LICENSE file in root directory.
 # Copyright(c) 2022 Intel Corporation. All Rights Reserved.
 
+include(${CMAKE_CURRENT_LIST_DIR}/topics/device-info/device_info.cmake)
```

Comment:
I think `device-info-topic-files.cmake` might be better as it corresponds to the variable name

Reply:
And here you want to include `dds_messages.cmake`, no?

Reply:
👍